configure.ac

```diff
@@ -174,7 +173,7 @@ PKG_CHECK_MODULES(DBUS,[
  dbus-1 >= $DBUS_REQUIRED
  gthread-2.0])
 
-PKG_CHECK_MODULES(MATE, [
+PKG_CHECK_MODULES(CAIRO, [
```

Comment:
This will require changing ```MATE_CFLAGS``` and ```MATE_LIBS``` to the corresponding ```CAIRO_xxx``` variables in Makefiles. Be careful :smile: